Samples/Client/SwiftClient/SwiftClient/Config.swift

```diff
@@ -0,0 +1,8 @@
+import WebRTC
+struct Config {
```

Comment:
we'll eventually need to rework this to use `webrtcConfig.json` but i'm ok punting that to a future work item.

Reply:
👍